File: 1PSP/ClienteRest/src/main/java/dao/modelos/BuscarSerie.java
package dao.modelos;

import java.util.List;
import com.google.gson.annotations.SerializedName;
import lombok.Getter;

@Getter
public class BuscarSerie{

	@SerializedName("page")
	private int page;

	@SerializedName("total_pages")
	private int totalPages;

	@SerializedName("results")
	private List<SerieSacadaDeBuscar> results;

	@SerializedName("total_results")
	private int totalResults;
}

File: 2Moviles/PelisYSeriesApp2_0/app/src/main/java/com/example/pelisyseriesapp/data/modelo/api/BuscarSerie.kt
package com.example.pelisyseriesapp.data.modelo.api


import com.google.gson.annotations.SerializedName

data class BuscarSerie(
    @SerializedName("page")
    val page: Int,
    @SerializedName("results")
    val resultadoBuscarSeries: List<ResultadoBuscarSerie>,
    @SerializedName("total_pages")
    val totalPages: Int,
    @SerializedName("total_results")
    val totalResults: Int
)